src/redux/authSlice.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from "@reduxjs/toolkit";
+
+const getUser = () => {
+  const existingUser = localStorage?.getItem("user");
+  if (existingUser) {
+    return JSON.parse(existingUser);
+  }
+  return null;
+};
+
+const initialState = {
+  user: getUser(),
+};
+
+const authSlice = createSlice({
+  name: "auth",
+  initialState,
+  reducers: {
+    login: (state, action) => {
+      state.user = action.payload;
```

Comment:
Could you please check out the state?  I am getting an empty state. Could you kindly fix it? I am unable to start and integrate with The appointment functions I am doing.
![Screenshot from 2023-09-04 14-15-01](https://github.com/sumon766/frontend-doctor-appointment/assets/86189460/a5e404d1-f0b9-4ff4-9aac-cae371cd98c3)